python/hopsworks_common/command.py

```diff
@@ -0,0 +1,59 @@
+#
+#   Copyright 2022 Hopsworks AB
```

Comment:
Since git recognizes this as a new file, can you also update the year?

Reply:
This seems to be a problem only with the GitHub view of the changes. If you switch to blame, it shows all the old commits.